File: netease-nimsdk/src/main/java/com/zxn/netease/nimsdk/business/session/module/IMultiRetweetMsgCreator.java
package com.zxn.netease.nimsdk.business.session.module;

import com.zxn.netease.nimsdk.api.model.CreateMessageCallback;
import com.netease.nimlib.sdk.msg.model.IMMessage;

import java.util.List;

public interface IMultiRetweetMsgCreator {
    void create(List<IMMessage> msgList, boolean shouldEncrypt,  CreateMessageCallback callback);
}

File: netease-nimsdk/src/main/java/com/zxn/netease/nimsdk/business/session/module/IMultiRetweetMsgCreator.kt
package com.zxn.netease.nimsdk.business.session.module

import com.netease.nimlib.sdk.msg.model.IMMessage
import com.zxn.netease.nimsdk.api.model.CreateMessageCallback

interface IMultiRetweetMsgCreator {
    fun create(msgList: List<IMMessage?>?, shouldEncrypt: Boolean, callback: CreateMessageCallback?)
}